grasshopper.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "1.1.4",
+    "description": "SSH client",
+    "homepage": "https://grasshopper.coding.kiwi",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://grasshopper.coding.kiwi/faq#free"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://grasshopper.coding.kiwi/download/1-1-4-grasshopper-Setup-1-1-4-exe#/dl.7z",
+            "hash": "d8950908ea59a08823d9faf1ea7dcfa74bdfbfcc59bf33fbbfca3db12ba71aaf",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "bin": "graSSHopper.exe",
```

Comment:
What are available parameters?

Reply:
None that I'm currently aware of. I can submit a new PR when some are added. I've been in conversation with the maintainer for a few for features that are desired.

Reply:
Then remove the bin property

Reply:
Sure, I don't understand why, but ok.